### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by HenaryKhuraijam**
# Changes made in this PR:
-  Created a new GitHub repo.
- Created a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
- Added a .gitignore file.
- Added node_modules and text.md to gitignore file.
- Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config).
- Added a descriptive README file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Project</title>
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <h1>Hello Microverse!</h1>
```
Comment (brainconnect93) [inline index.html]: - [ ] I would kindly suggest you give your `h1` a `class name` then the class name should be used to style your h1 content in the CSS file.

E.G:
```HTML
<h1 class="header">Hello Microverse</h1>
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: aqua;
+}
```
Comment (brainconnect93) [inline style.css]: - [ ] The class name that was given to your h1 in the HTML file should be used here to style the content

E.G:
``` CSS
.header {
    color: aqua;
  }
```

With this, you will be sure you follow the project requirement

File: README.md [hunk 3]
```diff
@@ -1 +1,64 @@
-# microverse
\ No newline at end of file
+# Hello Microverse
+
```
Comment (brainconnect93) [inline README.md]: - [ ] In your README file kindly add the `table of content` to it, and all the necessary headers with their respective content describing what they entail. headers include:

  - `Key Feature`
  - `Future Features`

Also kindly remove the live server link as you don't have any deployed link for your project now. You can simply replace the section with a text stating `link not available`.

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @HenaryKhuraijam, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (HenaryKhuraijam) [reply README.md]: Thank you for your suggestion, I will make the required changes.

**Comment by HenaryKhuraijam** [comment]
# Made Changes as suggested:
- Add class in h1 tag and using class name in css 
- Modified README file

File: README.md [hunk 4]
```diff
@@ -1 +1,69 @@
...
+> Hello Microverse is first project for microverse .
+
+## Built With
```
Comment (nahnahdev) [inline README.md:5]: - [ ] I noticed that you are still missing the Table of contents for the README file. You can find the table of contents here in [this template](https://github.com/microverseinc/readme-template) :+1: 
- [ ]  Just as the first reviewer suggested, Kindly add the Table of Contents as it is essential for navigation through your README file :100: 
- [ OPTIONAL ] Please note that all the sections of the [Microverse Readme template](https://github.com/microverseinc/readme-template) are compulsory with the exception of the (`FAQs` section). So I will suggest you include all the other sections you exempted from the README template and fill them with the right information even though this is an optional comment :100: 
- In this case, the section you are missing is the `Future Features` section

**Review by nahnahdev — CHANGES_REQUESTED** [review]
Hi @HenaryKhuraijam ,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
- [ ] Good use of GitHub flow
- [ ] Good PR title and description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] I noticed that in your files, you didn't properly include the linters file so the linter checks didn't run on Github as they should have. Currently, you have `.github/linter.yml` but it should be `.github/workflows/linters.yml`. So you're missing the second folder `workflows` which should be in the `.github` folder and the `linters.yml` file you were missing the `s` in yours should be placed in the `workflows` folder and after that you'll have this `.github/workflows/linters.yml`
- [ ] KIndly follow the steps stated in this repo [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) if you are still having trouble with properly setting it up :100: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 4 above)
Comment (HenaryKhuraijam) [reply README.md:5]: Thank you for finding out my mistakes, I will make the required changes.

**Comment by HenaryKhuraijam** [comment]
# Made Changes as suggested:
- Rename linters file
- Add Future Features in  README file

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hi @HenaryKhuraijam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ Implemented requested changes 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**